dotcom-rendering/src/web/lib/filterSwitches.ts

```diff
@@ -0,0 +1,6 @@
+export type ABTestSwitches = Record<`ab${string}`, boolean>;
+
+export const filterABTestSwitches = (switches: Switches): ABTestSwitches =>
+	Object.fromEntries(
+		Object.entries(switches).filter(([key]) => key.startsWith('ab')),
```

Comment:
Is there a convention that switches used for ab tests are given the `ab` prefix and this function is built on top of that pattern?

Reply:
Yes, this is [a convention followed in frontend](https://github.com/guardian/frontend/blob/016e9e268b47f2fef04f491af12d7e2c9577a2fe/common/app/conf/switches/ABTestSwitches.scala) and [the ab-testing framework expects it](https://github.com/guardian/ab-testing/blob/b5de0e09259d99399f192e4696470812b2aa45b1/packages/ab-core/src/core.ts#L29-L30).

Sadly I couldn’t find documentation outside of the actual LoC where this stuff is evaluated. I tried to bring a change to the ab-testing library to use a template string to indicate this fact with `ab${string}`, but couldn’t get the repo to behave.